r/data-raw/docgen.R

```diff
@@ -85,9 +85,7 @@ library(dplyr)
 library(purrr)
 
 # Functions that for whatever reason cause xref problems, so don't hyperlink
-do_not_link <- c(
-  "stringr::str_like" # Still only in the unreleased version
-)
+do_not_link <- c()
```

Comment:
Given this us now an empty vector, how about we remove this and update the code that uses it?

Reply:
That would be fine, but considering the possibility of similar cases occurring in the future, I thought it was worth leaving it for then.

Reply:
That's logical. The reason I suggested removing it is that these cases occur very infrequently if at all; this is the only example out of >200 functions and happened because we implemented a function from the dev version of stringr, which is highly unusual and there are no plans to do this again.

That said it's so minor, I'm not going to block merging because of it; let me know if you'd rather leave it in and I'll approve this PR.

Reply:
Thank you for the explanation. I don't know how `str_like` was implemented here, so your observation that the same is unlikely to happen in the future makes sense to me.

However, updating the script that generates the documentation seems generally more compelling to track in a separate PR, given that PR commits are squashed, so I would still prefer to keep it in this PR. (I think it would be far easier to revert from the future if this process were removed in another PR)

Reply:
That's reasonable, makes sense.